config/detekt/detekt.yml

```diff
@@ -118,7 +118,7 @@ complexity:
     threshold: 60
   LongParameterList:
     active: true
-    functionThreshold: 7
```

Comment:
As I said in the description since the Compose proposes more parameters than usual, I changed that value.  

Anyway, I propose to escalate that question in the next meeting, and together we can find the strategy.
But now I propose to change that value, to at least 8. 

Otherwise, it will lead to refactoring code in another place, and I'm not sure that is on the current scope.

Reply:
I agree with @TrubkaU, but next time it's better to discuss it with the team